### ISSUE TITLE
how to get 8-bit gray labels?

### ISSUE THREAD

**Issue by x7hkvip** [wkentaro/labelme#384]
I am totally new in labelme,I used python script labelme2voc.py generated 8-bit RGB labels in folder SegmentationClassPNG.But when I training ENet(tensorflow version),it needs 8-bit gray labels,what shoud i do?Could anyone else tells me how to modify the script?
Thanks a lot for your help!

**Comment by passion3394** [comment]
I think you can use just one channel of the RGB file.

**Comment by wkentaro** [comment]
But I think the files inside of `SegmentationClassPNG` is always 8-bit gray labels.

```
https://github.com/wkentaro/labelme/blob/7bef7f87f842eefc28d51f7064cd88ef1637f52f/labelme/utils/_io.py#L15-L19
```

**Comment by x7hkvip** [comment]
@wkentaro Very appreciate for your help!I use labelme to get json files of my dataset,then I used labelme2voc.py python script to generate label images.But the label images are RGB-8bit images,I checked the images info,it shows the images have three channels(Red 8bit,Green 8bit,Blue 8bit),I want the pixels in my label images equals my label,such as I have 9 classes,I want pixels during 1 to 9,what shoud i do? 
Thank you very much!
[2019-04-28 09-27-39屏幕截图](https://user-images.githubusercontent.com/40617232/56856913-e84f4c80-6997-11e9-816b-8c03f8874f12.png)

**Comment by x7hkvip** [comment]
Now I fixed the problem in a clunky way. QAQ
First,using labelme2voc.py get 8-bit RGB label images.
Second,use opencv to convert 8-bit RGB label images into gray imagres.Note that in this step,the pixel value of each classes is strange.Such as my 1 class should be 1 but 38.
Finally,I wrote a script to step through every pixels and assign a new number of my classes.
Now,I have 8-bit gray pixels and each pixels are during 0 to 9(cause I have 10 classes)
